my_view_page.php

```diff
@@ -89,13 +89,17 @@
 #print_r ($t_boxes);
 
 $t_project_id = helper_get_current_project();
+
+# Add class "col-md-7" only if the timeline is shown.
 ?>
-<div class="col-md-7 col-xs-12">
+<div class="col-xs-12 <?php echo ( access_has_project_level(config_get( 'timeline_view_threshold' )) ) ? "col-md-7" : "col-md-6"; ?>">
```

Comment:
Cache the config_get( 'timeline_view_threshold' ) in a $t_timeline_view_threshold variable rather than getting it twice within the page.  Here and when including the timeline information below.